config/settings.yml.sample

```diff
@@ -25,12 +25,12 @@ test_enable_bias_tee: true
 test_freq_offset: 0
 
 # receiver settings
-#   meteor_receiver - which receiver method to use (either 'rtl_fm' or 'gnuradio')
 #   noaa_receiver  - which receiver method to use (either 'rtl_fm' or 'gnuradio')
+#   meteor_receiver - which receiver method to use (either 'rtl_fm' or 'gnuradio')
 #     **WARNING**: 'gnuradio' does not work with certain SDR devices (e.g. it will not currently
 #                  work with a RTL-SDR v3 dongle, as no image will be decoded from the bitstream)
-meteor_receiver: 'gnuradio'
 noaa_receiver: 'gnuradio'
+meteor_receiver: 'rtl_fm'
```

Comment:
I did that since I've only so far integrated MeteorDemod into rtl_fm section since it seemed easier to me, I'll also integrate it into GNU Radio section. This is only for now.